components/Blocks/PlainHero.css

```diff
@@ -0,0 +1,30 @@
+.root {
+  background: var(--color-offwhite);
+  color: var(--color-grey);
```

Comment:
Again, mild concerns over using explicit colour variables anywhere outside of initially setting our functional variables up. (e.g. loading `--color-grey` into `--color-subtle` or whatever).